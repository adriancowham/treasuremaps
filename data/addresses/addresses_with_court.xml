<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339978699.html"> 
<title><![CDATA[ESTATE SALE (CARMICHAEL)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339978699.html</link> 
<description><![CDATA[Sale this weekend, April  22 and 23.  Sale will be from 9am to 4 pm.  1825 St. Ann Ct. Carmichael 95608 near Arden and Fair Oaks Blvd.
<br> 

<br> 
Great sale, don't miss it.
<br> 

<br> 
Very nice furniture  much of it antique.  Books, records,  costume jewelry, sewing, nice old linens, hundreds of buttons, marbles, toys, primitives, great old wooden boxes, c.1890 roll top desk, lots of picture frames, lots of beautiful dishes, bisque kewpies, japanese dolls,  like new Shop Smith, great work bench, tools, wonderful flower pots, patio tables, king size bed, piano, platform rocker, lots and lots of Christmas, both old and new.  Childrens books.  Side by side fridge, and a 1950's running  fridge.  Occupied Japan items, prints, picture frames.  Hundreds and hundreds of items.
<br> 

<br> 

<br> 
 
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=CARMICHAEL -->Location: CARMICHAEL
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:55:39-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339978699.html</dc:source> 
<dc:title><![CDATA[ESTATE SALE (CARMICHAEL)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:55:39-07:00</dcterms:issued> 
</item> 
</posts>