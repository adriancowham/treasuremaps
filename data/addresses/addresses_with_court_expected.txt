1825 St. Ann Ct.